packages/admin/addon/controllers/admin/query-stats.ts

```diff
@@ -5,15 +5,11 @@
 
 import Controller from '@ember/controller';
 
-export default class AdminQueryStats extends Controller.extend({
-  // anything which *must* be merged to prototype here
-}) {
-  // normal class body definition here
-}
+export default class AdminQueryStatsController extends Controller {}
 
 // DO NOT DELETE: this is how TypeScript knows how to look up your controllers.
 declare module '@ember/controller' {
   interface Registry {
-    'admin/query-stats': AdminQueryStats;
+    'query-stats': AdminQueryStatsController;
```

Comment:
Can we just leave this as `admin/query-stats` for now, and also update users to `admin/users` since that's what the blueprint creates